### PR TITLE
EXL-318 General fixes

### PR DESCRIPTION
**Pull request by zsso**

### CODE REVIEW

File: plugin/inc/intorder.class.php [hunk 1]
```diff
@@ -144,7 +146,7 @@ public function showForm($ID, $options = [])
             $consumable = new PluginIserviceConsumable();
             $consumable->getFromDB($consumable_id);
             $intorders              = new PluginIserviceIntOrder();
-            $other_orders           = PluginIserviceCommon::getQueryResult(
+            $other_orders           = PluginIserviceDB::getQueryResult(
```
Comment (hupupu) [inline plugin/inc/intorder.class.php:149]: Won't this require a `use`?

File: plugin/inc/pendingemailupdater.class.php [hunk 2]
```diff
@@ -44,7 +44,7 @@ static function cronupdatePendingEmails($task)
     {
         global $DB, $CFG_PLUGIN_ISERVICE;
 
-        if (empty($CFG_PLUGIN_ISERVICE['enabled_crons']['updatePendingEmails'])) {
+        if (empty(PluginIserviceConfig::getConfigValue('enabled_crons.updatePendingEmails'))) {
```
Comment (hupupu) [inline plugin/inc/pendingemailupdater.class.php:47]: `use`?

File: plugin/inc/pendingemailupdater.class.php [hunk 3]
```diff
@@ -55,7 +55,7 @@ static function cronupdatePendingEmails($task)
             $task->log("Invalid path for dbf files: $dbf_path_base\n");
         }
 
-        $pending_emails = PluginIserviceCommon::getQueryResult(
+        $pending_emails = PluginIserviceDB::getQueryResult(
```
Comment (hupupu) [inline plugin/inc/pendingemailupdater.class.php:58]: `use`?

**Review by hupupu — DISMISSED** [review]

File: plugin/inc/intorder.class.php (hunk 1 above)
Comment (zsso) [reply plugin/inc/intorder.class.php:149]: I think it is not needed, if I insert it I got this:

image

File: plugin/inc/pendingemailupdater.class.php (hunk 2 above)
Comment (zsso) [reply plugin/inc/pendingemailupdater.class.php:47]: Same as above.

**Pull request merged**